test/PresentationExchange.spec.ts

```diff
@@ -95,6 +97,50 @@ async function getPayload(): Promise<AuthenticationRequestPayload> {
   };
 }
 
+async function getPayload_PD_Ref(): Promise<AuthenticationRequestPayload> {
```

Comment:
Why are we all of a sudden using a different naming convention (camelcase/snakecase)?